addons/website_event/models/website_visitor.py

```diff
@@ -69,7 +69,7 @@ def _search_event_registered_ids(self, operator, operand):
         'in', [1, 2])] should return visitors having a registration on events 1, 2 as
         well as their children for notification purpose. """
         if operator == "not in":
-            raise NotImplementedError("Unsupported 'Not In' operation on visitors registrations")
+            raise NotImplementedError(_("Unsupported 'Not In' operation on visitors registrations"))
```

Comment:
Hey @HydrionBurst 

About `NotImplementedError`:
It can be trigger by UI :thinking: When user is using studio or adding domain in search view
So, IMO it should be translated :innocent: 
What do you think? :pleading_face: 
If is not really useful then do we have to remove it from everywhere? :thinking: 

Thank you :slightly_smiling_face:

Reply:
@tde-banana-odoo :arrow_up: 
What do you think? :thinking: